Makefile

```diff
@@ -0,0 +1,15 @@
+UK_ROOT ?= $(PWD)/.unikraft/unikraft
+UK_LIBS ?= $(PWD)/.unikraft/libs
+
+LIBS-y :=
```

Comment:
Why did you put the libraries like so and not on the same line?

Reply:
No particular reason, it was easier to remove and add libraries that way 😆